### PR TITLE
Run build harness as gladly.

### PR DESCRIPTION
**Pull request by caileighadams**
Add user to build harness image.

### CODE REVIEW

**Comment by caileighadams** [comment]
not ready for merge

File: Dockerfile [hunk 1]
```diff
@@ -27,6 +27,9 @@ ENV KUBECTL /usr/local/bin/kubectl
 ENV KUBEUTIL $BUILD_HARNESS_PATH/kube-util
 WORKDIR $BUILD_HARNESS_PATH
 
+ENV USER gladly
```
Comment (mkj28) [inline Dockerfile]: I think you also need a line like:
`RUN addgroup -S $USER && adduser -S $USER $USER`?

Comment (caileighadams) [reply Dockerfile]: Yep.... missing that feels silly....

**Comment by nateg-gladly** [comment]
Seems reasonable to me! Wanted to just double-check: this repo is public, we're OK w/ having this instance of our Docker username in here, and also our `gladly.qa` domain names?

**Pull request merged**